Misc/NEWS.d/next/Core and Builtins/2021-12-05-17-36-08.bpo-45711.3TmTSw.rst

```diff
@@ -0,0 +1 @@
+:c:func:`_PyErr_ChainStackItem` no longer normalized ``exc_info`` (including setting the traceback on the exception instance) because ``exc_info`` is always normalized.
```

Comment:
```suggestion
:c:func:`_PyErr_ChainStackItem` no longer normalizes ``exc_info`` (including setting the traceback on the exception instance) because ``exc_info`` is always normalized.
```